incubator/java-openliberty/templates/binary/pom.xml

```diff
@@ -0,0 +1,149 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>dev.appsody.starter.java-openliberty</groupId>
+    <artifactId>tamigration</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <file.repo.name>migrationassets</file.repo.name>
+    </properties>
+
+    <repositories>
+        <!--local migration assets repo-->
+        <repository>
+            <id>project.local</id>
+            <name>project</name>
+            <url>file:${project.basedir}/${file.repo.name}</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <!-- example of adding a war file dependency -->
+        <!--
+        <dependency>
+			<groupId>com.mywar</groupId>
+			<artifactId>modresorts</artifactId>
+			<version>1.0</version>
+			<type>war</type>
+		</dependency>	
+    -->
+    </dependencies>
+
+    <build>
+        <finalName>${project.artifactId}</finalName>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.1.1</version>
+                <executions>
+                    <execution>
+                        <id>copy-war-dependency</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <!-- put war dependency file in target after dependency resolution -->
+                            <outputDirectory>${project.build.directory}</outputDirectory>
+                            <overWriteReleases>false</overWriteReleases>
+                            <overWriteSnapshots>true</overWriteSnapshots>
+                            <excludeTransitive>true</excludeTransitive>
+                            <includeTypes>war</includeTypes>
+                        </configuration>
+                    </execution>
+                    <!-- example of how to move a resource adapter dependency
+                    <execution>
+                        <id>copy-resource-adapter-artifacts</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/liberty/wlp/usr/servers/shared/resources</outputDirectory>
+                            <overWriteReleases>false</overWriteReleases>
+                            <overWriteSnapshots>true</overWriteSnapshots>
+                            <excludeTransitive>true</excludeTransitive>
```

Comment:
How did we come up with all these config parm values?  

I think the most helpful one might simply be:
```
      <includeGroupIds>mysql</includeGroupIds>
```

Besides that maybe a more helpful thing would be a link to the doc?  

One gotcha is if you're iterating through a 1.0 release testing this out (instead of using a snapshot), then overwrite false can prevent changes.   OK, maybe you should know better... I'd just as soon step aside altogether if we don't have a good reason for suggesting these values).

Reply:
i have updated the config of these stanzas to reflect the need for <includeGroupIds> ... i also detail <includeArtifactIds> as well